### ISSUE TITLE
Bug: AppleScript "cannot covert string to number" error on macOS 12.6

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/PhotoScript#38]
Interesting! Quite different on my machine... 
I'm running: 
```
❯ sw_vers
ProductName:	macOS
ProductVersion:	12.6
BuildVersion:	21G115
```
<img width="699" alt="Screenshot 2022-11-13 at 18 22 54" src="https://user-images.githubusercontent.com/3964888/201535191-ce9b75e0-d8d4-46f9-a7ef-60de65900a31.png">

_Originally posted by @AlexisMaizel in https://github.com/RhetTbull/osxphotos/discussions/828#discussioncomment-4129323_

**Comment by RhetTbull** [comment]
Actually, the problem is one of locale where `,` is used as decimal separator instead of `.`

**Comment by RhetTbull** [comment]
Fixed in 0.2.1
